ang/civicase/dashboard/directives/dashboard.directive.html

```diff
@@ -29,22 +29,32 @@ <h1 class="hidden" crm-page-title crm-document-title="myCasesOnly ? ts('CiviCase
             ng-model="filters.caseRelationshipType" />
           <a ng-if="checkPerm('add cases') && caseTypeCategoryName == defaultCaseCategory && !newCaseWebformUrl"
              ng-href="{{ url('civicrm/case/add', {reset: 1, action: 'add', context: 'standalone'}) }}"
-             class="btn btn-primary crm-popup civicase__dashboard__add-case"
+             class="btn btn-primary crm-popup civicase__dashboard__action-btn"
              crm-popup-form-success="refresh()">
             <i class="material-icons">add_circle</i>{{ ts('Add Case') }}
           </a>
           <a ng-if="checkPerm('add cases') && caseTypeCategoryName != defaultCaseCategory && !newCaseWebformUrl"
              ng-href="{{ url('civicrm/case/add', {reset: 1, action: 'add', case_type_category: caseTypeCategoryName, context: 'standalone'}) }}"
-             class="btn btn-primary crm-popup civicase__dashboard__add-case"
+             class="btn btn-primary crm-popup civicase__dashboard__action-btn"
              crm-popup-form-success="refresh()">
             <i class="material-icons">add_circle</i>{{ ts('Add Case') }}
           </a>
           <a ng-if="checkPerm('add cases') && newCaseWebformUrl"
```

Comment:
As now we can add buttons using the newly added Provider. Please move the existing buttons to the same approach.
Also in the existing code, we have the "Add Case" button 3 times, all with different `ng-if` logic. 
So lets refactor it and have a single button, and all the ng-if should be moved to `isVisible` function logic.

PS: I thought I mentioned this in the ticket, as there was a discussion between myself and @tunbola . But looks like I forgot. Apologies.

Reply:
That is a great idea @deb1990